src/content/update_manager.cc

```diff
@@ -177,8 +177,7 @@ void UpdateManager::threadProc()
             }
             bool sendUpdates = true;
             if (sleepMillis >= MIN_SLEEP && objectIDHash->size() < MAX_OBJECT_IDS) {
-                std::chrono::milliseconds timeout;
-                getTimespecAfterMillis(sleepMillis, timeout);
+                auto timeout = getTimespecAfterMillis(sleepMillis);
 
```

Comment:
Seems this call is not needed altogether.

Reply:
Funny. Now this functions is only used in one location. Might make sense to remove and just move it to updateNextNotify

Reply:
Looking at the complexity of it -> do it